at_cram/bin/at_cram_single.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:crypto/crypto.dart';
+
+// command-line usage
```

Comment:
Should this section be using /// for doc comments?

Reply:
See 0724d3f3a1c5d46a5ee43d97e97886b7164fbe97

Is that correct?